OSDOCS-1142: Adding priority and namespace filtering (#26224)
openshift/openshift-docs: modules/nodes-descheduler-filtering-namespace.adoc

// Module included in the following assemblies:
//
// * nodes/scheduling/nodes-descheduler.adoc

[id="nodes-descheduler-filtering-namespace_{context}"]
= Filtering pods by namespace

> You can configure whether or not pods should be considered for eviction based on their namespace. Only the following descheduler strategies support namespace filtering:

Reviewer: ```suggestion
You can configure whether or not pods are considered for eviction based on their namespace. Only the following descheduler strategies support namespace filtering:
```

* `PodLifeTime`
* `RemoveDuplicates`
* `RemovePodsHavingTooManyRestarts`
* `RemovePodsViolatingInterPodAntiAffinity`
* `RemovePodsViolatingNodeAffinity`
* `RemovePodsViolatingNodeTaints`

> 

Reviewer 2: @bergerhoffer  can we remove `RemoveDuplicates` and add `RemovePodsHavingTooManyRestarts`
RemoveDuplicates and LowNodeUtilization does not support NamespaceFiltering.

Author: @kasturinarra I got this list from here, and it lists `RemoveDuplicates`: https://github.com/kubernetes-sigs/descheduler#namespace-filtering

Does OpenShift's descheduler differ? CC @damemi 

Also `RemovePodsHavingTooManyRestarts` is already in the list.

Author: @damemi Confirmed with me that `RemoveDuplicates` was only recently added, and so not a part of 4.6. Removing!

Reviewer 2: Thanks !!
